apps/server/src/modules/common-cartridge/import/common-cartridge-file-parser.ts

```diff
@@ -0,0 +1,22 @@
+import AdmZip from 'adm-zip';
+import { CommonCartridgeManifestParser } from './common-cartridge-manifest-parser';
+
+export class CommonCartridgeFileParser {
+	public readonly manifest: CommonCartridgeManifestParser;

```

Comment:
rename manifest to manifestParser, it is somehow clearer.

Reply:
Done.